pysages/ssages/methods/test/string_method.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+
+import sys
+from mpi4py import MPI
+import os
+os.environ["XLA_PYTHON_CLIENT_MEM_FRACTION"]= ".{0}".format(int(90/MPI.COMM_WORLD.Get_size()))
+import jax
+import jax.numpy as np
+from pysages.ssages.methods.string_method import interpolate_cv
+import matplotlib.pyplot as plt
+
+
+def create_circle(radius, size):
+    image = np.zeros((size, size))
+    for x in range(size):
+        for y in range(size):
+            if (x-size/2)**2 + (y-size/2)**2 < radius**2:
+                image = jax.ops.index_add(image, (x, y), 1)
```

Comment:
The recommended (but equivalent) way to this in jax is

```suggestion
                image = image.at[(x, y)].add(1)
```